### PR TITLE
Add exception logging

### PR DESCRIPTION
**Pull request by ameech**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -13,12 +13,13 @@
         "php": ">=5.5",
         "arbiter/arbiter": "^0.1",
         "destrukt/destrukt": "^0.7.1",
+        "equip/adr": "^1.0",
         "filp/whoops": "^1.1",
         "nikic/fast-route": "^0.7",
+        "monolog/monolog": "^1.17",
```
Comment (shadowhand) [inline composer.json]: Shouldn't this be a PSR require? With a `suggest` for monolog?

Comment (ameech) [inline composer.json]: Sure can, but will need to fix this file: https://github.com/equip/framework/pull/4/files#diff-4f99c1eaf57e677d457910a90234d70dR7

Comment (shadowhand) [inline composer.json]: Nah we don't have to change anything there. We do a similar thing with Plates already.

Comment (ameech) [inline composer.json]: Updated.

File: composer.json [hunk 2]
```diff
@@ -13,18 +13,20 @@
...
         "willdurand/negotiation": "^2.0",
         "zendframework/zend-diactoros": "^1.0.4"
     },
     "require-dev": {
         "josegonzalez/dotenv": "^1.2",
         "league/plates": "^3.1",
+        "monolog/monolog": "^1.17",
         "phpunit/phpunit": "^4.8|^5.0"
     },
     "suggest": {
```
Comment (shadowhand) [inline composer.json]: Add a suggestion to monolog too so that users know they need to install monolog to use the logger.

Comment (shadowhand) [inline composer.json]: I would also add a PR to equip/project that requires monolog there, since we'll need one by default.

Comment (ameech) [inline composer.json]: Updated, here's the project PR: https://github.com/equip/project/pull/2

File: src/Handler/ExceptionHandler.php [hunk 3]
```diff
@@ -84,6 +93,10 @@ public function __invoke(
             $handler = $this->handler($type);
             $this->whoops->pushHandler($handler);
 
+            $this->whoops->pushHandler(function($exception) {
+                $this->logger->error($exception);
```
Comment (shadowhand) [inline src/Handler/ExceptionHandler.php]: I realize this is really late, but couldn't we avoid changing this interface by adding the logging handler in the configuration? There's nothing that specifically needs to have this, other than Whoops, and Whoops is already configured elsewhere.

**Comment by elazar** [comment]
@ameech This will need to be modified to be merged. The configuration in `ExceptionLoggerConfiguration` from this PR should go into `WhoopsConfiguration`, else it will overwrite it in the injector if the two classes are both used at the same time.

**Comment by ameech** [comment]
@elazar yep! This became stagnant after we got stuck on how to configure it exactly. I'll check it out again this weekend.

**Pull request closed without merging**